packages/manager/apps/key-management-service/src/components/credential/create/identities/list/IdentitiesServiceAccountList.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Dispatch, Fragment, SetStateAction, useState } from 'react';
+import { OsdsSearchBar, OsdsDivider } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { IdentityOauthClient } from '@/types/identity.type';
+import IdentitiesServiceAccountTile from '../tile/IdentitiesServiceAccountTile.component';
+import identityListSortAndFilter from '@/utils/credential/identityListSortAndFilter';
+
+type IdentitiesServiceAccountProps = {
+  serviceAccountList: IdentityOauthClient[];
+  selectedServiceAccount: IdentityOauthClient[];
+  setSelectedServiceAccount: Dispatch<SetStateAction<IdentityOauthClient[]>>;
+};
+
+const IdentitiesServiceAccountList = ({
+  serviceAccountList,
+  selectedServiceAccount,
+  setSelectedServiceAccount,
+}: IdentitiesServiceAccountProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+  const [
+    sortedFilteredServiceAccount,
+    setSortedFilteredServiceAccount,
```

Comment:
**nitpick**: `Accounts` or `AccountList` plural as they're arrays (for selected/setSelected/sorted/setSorted accounts)